test/test_examples_2d_parallel.jl

```diff
@@ -11,6 +11,7 @@ Trixi.mpi_isroot() && isdir(outdir) && rm(outdir, recursive=true)
 
 # pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
 const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
+const TEST_EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "test", "examples", "2d")
```

Comment:
```suggestion
```
Unused?

Reply:
Well, that explains why the tests are failing.

Reply:
Why don't you remove it then?